google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITAddressesTest.java

```diff
@@ -120,4 +103,36 @@ public void testAggregatedList() {
     }
     Assert.assertTrue(presented);
   }
+
+  @Ignore("Non ascii symbols are not converted correctly")
```

Comment:
Of there is an issue opened to edit this case, please put a TODO with the link to the issue here (or create the issue first, and then add the link).

Reply:
Done